README.md

```diff
@@ -0,0 +1,34 @@
+# Wicked Migration
+
+Provides a webserver for migrating Wicked configuration files to NetworkManager configuration files.
+
+### Build:
+
+`cargo build --release`
+
+### Run Server:
+
+Options
+| Option | Description |
+|--------|-------------|
+| -s     | Specify where static files are located |
+| -i     | Set IP address |
+| -p     | Set port |
+| -d     | Set location for the database |
+| -h     | Show help information |
+
+Configuration files are automatically deleted after 5 minutes or upon retrieval.
+
+### Migration:
+
+Migration can be accessed through a web frontend or the commandline:
+
+#### Commandline:
+
+multiple files:
+
+`curl -v -L POST -H  "Content-Type: application/xml" --data-binary "@/path/to/eth0.xml" <ipaddr>:<port> --output /path/to/output.tar`
```

Comment:
This doesn't work, I think you forgot the `-X` before `POST`, but that is implicit with `--data-binary` anyway, so you could remove it or keep it explicit, as you like.
Also since most people don't have a xml with their config lying around I would maybe add a command that doesn't take a file but instead directly uses the output of `wicked show-config`